app/src/responsive.sass

```diff
@@ -1 +1,2 @@
 $breakpoint-tablet: 500px
+$breakpoint-width: 414px
```

Comment:
Not sure this breakpoint is right. Here is how it looks with a width of 415px:
![image](https://user-images.githubusercontent.com/1835665/121051285-a1a8bf00-c7b9-11eb-8a6c-2e01d34cfbb0.png)

Maybe you could just use `$breakpoint-table`. 

Anyway, the name `$breakpoint-width` is wrong, because:
1. All breakpoints will depend on the width of the screen only. It could be a good idea to add a comment at the start of this file to explain that.
2. It does not clearly relate to the other variable `$breakpoint-tablet`. If I read both variables used in some file, I cannot guess which one has a greater value than the other. I suggest `$breakpoint-smartphone` for instance.